lms/djangoapps/hera/models.py

```diff
@@ -342,3 +342,37 @@ def reset_user_onboarding(sender, instance, *args, **kwargs):
     cache.delete(
         CACHING_USER_ONBOARDING_TEMPLATE.format(user_id=instance.user_id, model_name=sender.__name__)
     )
+
+
+class Medal(models.Model):
+    title = models.CharField(max_length=100)
+    image = models.ImageField(upload_to='hera/medals')
+    dna_icon = models.ImageField(upload_to='hera/medals', verbose_name="DNA icon")
+    min = models.PositiveSmallIntegerField()
+    max = models.PositiveSmallIntegerField()
+
+    def __str__(self):
+        return self.title
+
+    @classmethod
+    def _instance(cls):
+        return cls.objects.all().order_by('min')
+
+    @classmethod
+    def user_dashboard_medals_data(cls):
+        instance = cls._instance()
+        if instance:
+            return instance
+
+    @classmethod
+    def get_medal(cls, points):
+        instance = cls._instance()
+        medal_title = ''
+        medal_dna_icon = ''
+        if instance:
+            for medal in instance:
+                if medal.min <= points <= medal.max:
```

Comment:
is it possible to you the django lookups `min__gte=something` - or something like that?

Reply:
Done